tests/non-js-resources/subtests/html/subtests/preload-assets/parcel/index.md

```diff
@@ -1,6 +1,5 @@
 ---
-result: fail
-issue: https://github.com/parcel-bundler/parcel/issues/3757
+result: pass
```

Comment:
_chef kiss_ This is pretty amazing.

One question tho: It seems that parcel adds `<link rel=stylesheet>` tag to the HTML because of the `import './styles.css';` in the JS file. That seems reasonable. However, it also puts that tag into the static markup with this code:

```js
document.onload = () => import('./styles.css');
```

Is that intentional? A bug?

Reply:
Where are you seeing this in the output? I tried changing the `import './styles.css'` to the above code, and I no longer get the `<link>` tag in the HTML.